package.json

```diff
@@ -9,7 +9,8 @@
     "watch:dev": "nodemon",
     "lint": "npx eslint .",
     "lint:fix": "npx eslint --fix .",
-    "format": "npx prettier --write \"**/*.+(js|jsx|json|yml|yaml|css|md)\"",
+    "format:win": "npx prettier --write \"**/*.{js,jsx,json,yml,yaml,css,md}\"",
```

Comment:
nice 👍

Reply:
that is interesting, I'm glad it's fixed now